src/constants.py

```diff
@@ -167,3 +167,12 @@
 PRODUCT_DOCS_PERSIST_DIR = "./vector-db/ocp-product-docs"
 SUMMARY_INDEX = "summary"
 SUMMARY_DOCS_PERSIST_DIR = "./vector-db/summary-docs"
+
+# cache constants
+IN_MEMORY_CACHE = "in-memory"
+IN_MEMORY_CACHE_MAX_ENTRIES = 1000
+REDIS_CACHE = "redis"
+REMOTE_CACHE_HOST = "redis-stack.ols.svc"
```

Comment:
not critical at the moment but for the future this should probably be configurable so devs can stand up a redis service on their laptop and point to it.

of course they can use /etc/hosts to make it work, but it would be nicer if they could just set it to localhost via an env.

Reply:
Yes I had this as a #TODO in my mind while I was writing the code. Should have dropped a comment over there, my bad 🙁